workshops/build-a-binderhub/workshop-presentations/zero-to-binderhub.md

```diff
@@ -457,19 +448,22 @@ helm repo update
 
 Next, install the required Helm chart using the config files we created in [Step 2: Run `setup.sh`](#2-run-setupsh).
 
-```
-helm install jupyterhub/binderhub \
+```bash
+helm install binderhub jupyterhub/binderhub \
     --version=0.2.0-f565958 \
-    --name=binderhub \
     --namespace=binderhub \
     -f secrets/secret.yaml \
-    -f secrets/config.yaml
+    -f secrets/config.yaml \
+    --create-namespace
 ```
+
 * `--version` refers to the version of the BinderHub Helm Chart.
   Available versions can be found [here](https://jupyterhub.github.io/helm-chart/#development-releases-binderhub).
   We have used the version released on 11 August 2019.
-* `--name` and `--namespace` may be different, but it's recommended they be the same to avoid confusion.
+* It is recommended that `--namespace` be the same as the provided `NAME` in order to avoid confusion.
```

Comment:
```suggestion
* It is recommended that `--namespace` be the same as the provided `NAME` in order to avoid confusion.
  In this case, our `NAME` is `binderhub`.
```